_includes/scripts.html

```diff
@@ -46,11 +46,11 @@
   </div>
   <div class="filters-panel cf">
     <label class="filter-label" >Popular tags:</label>
-    <ul class="popular-tags">
+    <div class="popular-tags">
     <% _.each(popularTags, function(entry, key){ %>
-      <li><a><%-entry.name%></a> (<%-entry.frequency%>)</li>
+      <a title="Popular Tag: <%-entry.name%>" tabindex="0"><%-entry.name%> <span class="popular-tags__frequency">(<%-entry.frequency%>)</span></a>
```

Comment:
🤔 Can we do these styling changes without losing the `ul`/`li` semantics of these elements?